cinema/src/main/java/davidCinema/cinema/Service.java

```diff
@@ -0,0 +1,35 @@
+package davidCinema.cinema;
+
+
+import java.util.HashMap;
+
+public class Service {
+	
+	static int idCounter = 3;
+	//ArrayList<Movie> arrayList = new ArrayList<Movie>();
+	private HashMap <Integer, Movie> movieMap = new HashMap<>();
+	
+public void setUpMovies() {

```

Comment:
Could be placed in an empty constructor, either as a method call or the code itself

Reply:
this was John